pypeit/core/arc.py

```diff
@@ -541,19 +545,23 @@ def detect_lines(censpec, nfitpix=5, sigdetect = 10.0, FWHM = 10.0, cont_samp =
     arc = detns - cont_now
     pixt = detect_peaks(arc, mph=thresh, mpd=3.0) #, show=debug)
     # Gaussian fitting appears to work better on the non-continuum subtracted data
-    tampl, tcent, twid, centerr = fit_arcspec(xrng, arc, pixt, nfitpix)
+    tampl_fit, tcent, twid, centerr = fit_arcspec(xrng, arc, pixt, nfitpix)
     #tampl, tcent, twid, centerr = fit_arcspec(xrng, arc_in, pixt, nfitpix)
-
+    tampl_true = np.interp(pixt, xrng, detns)
+    tampl = np.interp(pixt, xrng, arc)
     #         sigma finite  & sigma positive &  sigma < FWHM/2.35 & cen positive  &  cen on detector
     # TESTING
-    good = (np.invert(np.isnan(twid))) & (twid > 0.0) & (twid < FWHM/2.35) & (tcent > 0.0) & (tcent < xrng[-1]) & (tampl < nonlinear_counts)
+    good = (np.invert(np.isnan(twid))) & (twid > 0.0) & (twid < FWHM/2.35) & (tcent > 0.0) & (tcent < xrng[-1]) & \
```

Comment:
Can you add an additional comment on what all is being 'tested'?
Thanks

Reply:
Done.